packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VXZoom/VXZoom.js

```diff
@@ -7,22 +7,29 @@ import ZoomEventLayer from '../ZoomEventLayer'
 class VXZoom extends PureComponent {
   constructor (props) {
     super(props)
-    const { setOnZoom } = props
+    const { setOnPan, setOnZoom } = props
 
+    setOnPan(this.handleToolbarPan.bind(this))
     setOnZoom(this.handleToolbarZoom.bind(this))
 
     this.onDoubleClick = this.onDoubleClick.bind(this)
+    this.onKeyDown = this.onKeyDown.bind(this)
     this.onMouseEnter = this.onMouseEnter.bind(this)
     this.onMouseLeave = this.onMouseLeave.bind(this)
+    this.onPan = this.onPan.bind(this)
   }
 
   static zoom = null
 
+  handleToolbarPan (type) {
+    this.onPan(type)
+  }
+
   handleToolbarZoom (type) {
     const doZoom = {
-      'zoomin': this.zoomIn.bind(this),
-      'zoomout': this.zoomOut.bind(this),
-      'zoomto': this.zoomReset.bind(this)
+      zoomin: this.zoomIn.bind(this),
```

Comment:
You don't need to bind this each time the handler runs. I think you can bind these once in the component constructor, then refer to the bound functions here.

Reply:
This is a copy of how it's setup elsewhere. It can be changed, but I think out of scope of this PR.

Reply:
Sure, but worth keeping an eye on this for memory leaks if `handleToolbarZoom` is called a lot.